apps/nextjs-website/tsconfig.json

```diff
@@ -16,7 +16,8 @@
       }
     ],
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./src/*"],
+      "@react-components/*": ["../../packages/react-components/dist/components/*"]
```

Comment:
since we are referencing a path of a package, is it possible to define it in the tsconfig in the root of the project?

Reply:
Tried several times, but unsuccesfully. To have this "path" we need to use it directly inside the tsconfig.json of next.js folder otherwise we'll have this error here: `Cannot find module '@react-components/Accordion' or its corresponding type declarations.`